terraform/instances.tf

```diff
@@ -397,6 +397,17 @@ resource "aws_instance" "worker" {
     ]
   }
 
+  provisioner "remote-exec" {
+    inline = [
+      "sudo mkdir -p /home/ubuntu/.hab/accepted-licenses"
```

Comment:
@eeyun, do you not need trailing commas here? My VSCode parser for Terraform complains if they are missing, as does Terraform...